### PR TITLE
Markdown editing notes

### PR DESCRIPTION
**Pull request by ravishende**
Added markdown editing functionality (still optional) in notes

### CODE REVIEW

File: src/notes/notes.js [hunk 1]
```diff
@@ -152,14 +120,11 @@ function _deleteCurrentNote(event) {
     // this allows us to have _deleteCurrentNote be a function that doesn't take in any parameters
     // so that we can use it with addEventListener and deleteEventListener to add/remove it.
     setTimeout(() => {
-        _deleteFromStorage(CURRENT_NOTE_ID);
+        deleteFromStorage(CURRENT_NOTE_ID);
     }, 10);
 }
 
-/**Edit the current note (displays note editor popup)
- * To be used with add/remove event listeners
```
Comment (shambhavi-20) [inline src/notes/notes.js:160]: Why were these comments removed? Are these not of JSDocs format?

**Review by shambhavi-20 — CHANGES_REQUESTED** [review]
All the comments were changed from a JSDocs format back to normal comments. Please  change the format of the comments.

File: src/notes/notes.js (hunk 1 above)
Comment (ravishende) [reply src/notes/notes.js:160]: Added this back in. Will change the rest of the comments to JSDocs format in a new branch `cleanup-notes`

**Review by shambhavi-20 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**